### PR TITLE
N36: Release

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/ca894c36ee4eddba3f8627415b0d54fadf62cd46): https://rust-gamedev-preview.netlify.app/news/036

Everyone is invited to review!

Hope to merge this in a few hours if there're no objections.

_Part of #1098_

### CODE REVIEW

**Review by 64kramsystem — COMMENTED** [review]
Looks good to me!

I've noticed that the "Storm Engine" link (https://rust-gamedev-preview.netlify.app/news/036/github.com/mooman219/Storm) is a 404, but I don't know if this is significant or can be ignored.

File: content/news/036/index.md [hunk 1]
```diff
@@ -338,20 +364,61 @@ July's full weekly devlogs: "This Week In Veloren...":
...
 on games like Lost Ark and Diablo. The author has been writing about their experience
 with changing how they approach working on personal projects [here].
 It is built using a custom UDP based networking stack, a custom ECS,
-and custom skinned mesh renderer to maximize what the author could learn.
+and a custom skinned mesh renderer to maximize what the author could learn.
 
 [Agma]: github.com/TuckerBMorgan/Agma
 [@TuckerBMorgan]: https://twitter.com/T_B_Morgan
 [here]: https://medium.com/@tucker.bull.morgan/summoning-a-devil-544b130c8889
 [Storm Engine]: github.com/mooman219/Storm
```
Comment (64kramsystem) [inline content/news/036/index.md:379]: Missing the `https://` protocol (prefix). I can't change it on the fly!

**Review by 64kramsystem — APPROVED** [review]
:+1:

**Comment by RobDavenport** [comment]
Looks like a few of Gamercade's links are broken as well, they're linking to `https://rust-gamedev-preview.netlify.app/news/036/Gamercade-Console` etc etc

File: content/news/036/index.md [hunk 2]
```diff
@@ -482,7 +549,7 @@ After over half a year in development, Gamercade and related tools are ready
 for pre-alpha testing. This includes the [console](Gamercade-Console) itself,
```
Comment (RobDavenport) [inline content/news/036/index.md:549]: Here as well as the other paragraphs further down... Are they supposed to be formatted as [console][Gamercade-Console] ?

Comment (RobDavenport) [inline content/news/036/index.md:549]:
```suggestion
for pre-alpha testing. This includes the [console][Gamercade-Console] itself,
```

**Comment by AngelOnFira** [comment]
Spelling + Grammar fixes in https://github.com/ozkriff/rust-gamedev.github.io/pull/5

File: content/news/036/index.md [hunk 3]
```diff
@@ -482,7 +549,7 @@ After over half a year in development, Gamercade and related tools are ready
...
 as well as the [editor](Gamercade-Editor).
```
Comment (RobDavenport) [inline content/news/036/index.md:550]:
```suggestion
as well as the [editor][Gamercade-Editor].
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
A few recommendations, but I'll take another look once a new render is up :100:

File: content/news/036/index.md (hunk 2 above)
Comment (ozkriff) [reply content/news/036/index.md]: fixed, tnx!

File: content/news/036/index.md [hunk 4]
```diff
@@ -338,19 +364,60 @@ July's full weekly devlogs: "This Week In Veloren...":
...
 _Moving around_
 
 [Agma] by [@TuckerBMorgan] is a 3D game built in the [Storm Engine] that is based
 on games like Lost Ark and Diablo. The author has been writing about their experience
 with changing how they approach working on personal projects [here].
 It is built using a custom UDP based networking stack, a custom ECS,
-and custom skinned mesh renderer to maximize what the author could learn.
+and a custom skinned mesh renderer to maximize what the author could learn.
 
 [Agma]: github.com/TuckerBMorgan/Agma
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:376]:
```suggestion
[Agma]: https://github.com/TuckerBMorgan/Agma
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Just a few comments:

* There's another broken link in the Agma section.
* NES Bundler's section is slightly weirdly phrased, maybe amend to `NES Bundler is a NES ROM packaging tool by..`?

Any other issues I spotted have already been flagged up :)

**Review by AngelOnFira — APPROVED** [review]
LGTM :) :rocket:

**Pull request merged**